portfolio/src/main/webapp/style.css

```diff
@@ -33,6 +33,20 @@
 }
 
 #homelinks img {
-    width: 50px;
-    padding: 10px;
+  width: 50px;
+  padding: 10px;
+}
+
+#projectscontent {
```

Comment:
nit: use dashes/hypens to concatenate words for css selectors
https://google.github.io/styleguide/htmlcssguide.html#ID_and_Class_Name_Delimiters